### PR TITLE
Migrates scenario guide from ansible/ansible repo

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Related to https://github.com/ansible-community/antsibull/pull/255.

Do we need to rename the page `index.rst`? Or add an `index.rst` file?

Also needs a PR against ansible/ansible removing this content from there. 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/962

### CODE REVIEW

File: docs/docsite/rst/guide_aws.rst [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+   :ref:`list_of_collections`
+       Browse existing collections, modules, and plugins
+   :ref:`working_with_playbooks`
+       An introduction to playbooks
+   :ref:`playbooks_delegation`
+       Delegation, useful for working with loud balancers, clouds, and locally executed steps.
+   `User Mailing List <https://groups.google.com/group/ansible-devel>`_
+       Have a question?  Stop by the google group!
+   `irc.freenode.net <http://irc.freenode.net>`_
+       #ansible IRC chat channel
```
Comment (tremble) [inline docs/docsite/rst/guide_aws.rst:281]: Do we want to update this to #ansible-aws ?

File: docs/docsite/rst/guide_aws.rst [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    ---
+    ec2_access_key: "--REMOVED--"
+    ec2_secret_key: "--REMOVED--"
+
+Note that if you store your credentials in vars_file, you need to refer to them in each AWS-module. For example::
+
+    - ec2
+      aws_access_key: "{{ec2_access_key}}"
+      aws_secret_key: "{{ec2_secret_key}}"
+      image: "..."
```
Comment (tremble) [inline docs/docsite/rst/guide_aws.rst:49]: I guess we'll want to update this to recommend the use of module_defaults?

**Review by tremble — APPROVED** [review]
The change itself looks fine to me.


@jillr I've dropped a couple of comments inline - I don't think they should be part of this PR (in the interests of keeping it clean and simple), do you think they're worth following up on?

File: docs/docsite/rst/guide_aws.rst (hunk 2 above)
Comment (acozine) [reply docs/docsite/rst/guide_aws.rst:49]: Thanks for spotting that @tremble! The guide will need sundry updates to the content. The purpose of this PR is mainly to get it moved, and to serve as an example of how to move other Scenario Guides.

Comment (acozine) [reply docs/docsite/rst/guide_aws.rst:49]: @tremble - Ignore my comment - I just saw your overall review comment . . .

File: docs/docsite/rst/guide_aws.rst (hunk 1 above)
Comment (jillr) [reply docs/docsite/rst/guide_aws.rst:283]: Definitely

**Comment by tremble** [comment]
@acozine This PR's currently flagged as draft, should we wait on something before merging this?

**Comment by acozine** [comment]
@tremble it's waiting on the antsibull PR listed above. Once that is tested and merged, we can publish .rst docs from collections to docs.ansible.com.

**Comment by jillr** [comment]
@acozine will we need to duplicate this doc in community.aws?  If I'm understanding antsibull/pull/255 there will no longer be anything at https://docs.ansible.com/ansible/devel/scenario_guides/guide_aws.html and folks will have to look under the specific collection for the guides?  I don't love the idea of duplicating it, but I'm trying to think through what the best thing for our users is.

**Comment by acozine** [comment]
@jillr yes, the idea here is to move the Scenario Guides into the collections that provide the functionality described in the guides. The content would be published to docs.ansible.com with the collection-based module documentation. 

We probably want to keep this content in one and only one collection, so either amazon.aws or community. aws. What is the difference between them? Does the content (for example, the dynamic inventory plugin) exist in both?

**Comment by goneri** [comment]
Can I suggest to use Sphinx-Ansible (https://sphinx-ansible.readthedocs.io/en/latest/) to be sure all the tasks work as expected? I understand it can be a bit of work and this is really just a suggestion.

**Comment by jillr** [comment]
@acozine different modules in each.  Supported modules/plugins are in amazon.aws, and community content is in community.aws.  The general content in the user guide is the same for both collections (how to authenticate, etc).  It could really stand to be updated (we want to discourage use of the ec2 module for example) but I'm not sure if we want/need two different guides.  I'll think on that a bit.

**Comment by jillr** [comment]
> The change itself looks fine to me.
> 
> @jillr I've dropped a couple of comments inline - I don't think they should be part of this PR (in the interests of keeping it clean and simple), do you think they're worth following up on?

I've added a task for myself to do the prose updates as part of the 2.0 release plan.

**Comment by jillr** [comment]
recheck

**Comment by jillr** [comment]
recheck

**Review by jillr — APPROVED** [review]

**Pull request merged**